src/__test__/landmark-router.test.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+import faker from 'faker';
+import superagent from 'superagent';
+import { startServer, stopServer } from '../lib/server';
+import { pCreateCountryMock } from './lib/country-mock';
+import { pCreateLandmarkMock, pRemoveLandmarkMock } from './lib/landmark-mock';
+import './lib/test.env';
+
+const apiURL = `http://localhost:${process.env.PORT}/api/v1/landmarks`;
+
+describe('VALID request to the API', () => {
+  describe('/api/v1/landmarks', () => {
+    beforeAll(startServer);
+    afterAll(stopServer);
+    afterEach(pRemoveLandmarkMock());
```

Comment:
Line 16 breaks a lot of things. Change this to:

```
afterEach(pRemoveLannmarkMock)
```
`afterEach` expects a function, NOT a function invocation, which is what you're doing, hence, the test module spits an error.